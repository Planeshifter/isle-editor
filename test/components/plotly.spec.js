// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import Plotly from '@isle-project/components/plotly/main.js';


// TESTS //

describe( '<Plotly />', function test() {
	it( 'renders a Plotly visualization', () => {
		const plot = renderer.create(
			<Plotly
				data={[{
					values: [ 24, 7, 0.5 ],
					labels: [ 'English', 'Spanish', 'Other' ],
					type: 'pie'
						}]}
				layout={{ width: 300 }}
			/>
		).toJSON();
		expect( plot ).toMatchSnapshot();
	});
});
