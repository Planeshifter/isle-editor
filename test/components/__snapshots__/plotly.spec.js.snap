// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Plotly /> renders a Plotly visualization 1`] = `null`;
